arrow-version/_includes/_js-bottom-docs.html

```diff
@@ -1,7 +1,7 @@
 <!-- Jquery -->
 <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
 <!-- Arrow playground -->
-<script src="https://unpkg.com/arrow-playground@1" data-selector="[class^='language-kotlin']"></script>
+<!--<script src="https://unpkg.com/arrow-playground@1" data-selector="[class^='language-kotlin']"></script>-->
```

Comment:
We shouldn´t comment on this line, we need `arrow-playground` to compile the code in the documentation.